### PR TITLE
Removed the cookies from the remote node rules.

### PR DESCRIPTION
**Pull request by knobunc**
The rest of the ovs sdn rules had removed them in other patches.  This catches the other cases we hadn't yet changed.

### CODE REVIEW

File: plugins/osdn/ovs/controller.go [hunk 1]
```diff
@@ -346,12 +345,11 @@ func (c *FlowController) AddOFRules(nodeIP, nodeSubnetCIDR, localIP string) erro
...
 	glog.V(5).Infof("AddOFRules for %s", nodeIP)
-	cookie := generateCookie(nodeIP)
 	otx := ovs.NewTransaction(BR)
 
-	otx.AddFlow("table=1, cookie=0x%s, priority=100, tun_src=%s, actions=goto_table:5", cookie, nodeIP)
-	otx.AddFlow("table=8, cookie=0x%s, priority=100, arp, nw_dst=%s, actions=move:NXM_NX_REG0[]->NXM_NX_TUN_ID[0..31],set_field:%s->tun_dst,output:1", cookie, nodeSubnetCIDR, nodeIP)
-	otx.AddFlow("table=8, cookie=0x%s, priority=100, ip, nw_dst=%s, actions=move:NXM_NX_REG0[]->NXM_NX_TUN_ID[0..31],set_field:%s->tun_dst,output:1", cookie, nodeSubnetCIDR, nodeIP)
+	otx.AddFlow("table=1, cookie=0x0, priority=100, tun_src=%s, actions=goto_table:5", nodeIP)
+	otx.AddFlow("table=8, cookie=0x0, priority=100, arp, nw_dst=%s, actions=move:NXM_NX_REG0[]->NXM_NX_TUN_ID[0..31],set_field:%s->tun_dst,output:1", nodeSubnetCIDR, nodeIP)
+	otx.AddFlow("table=8, cookie=0x0, priority=100, ip, nw_dst=%s, actions=move:NXM_NX_REG0[]->NXM_NX_TUN_ID[0..31],set_field:%s->tun_dst,output:1", nodeSubnetCIDR, nodeIP)
```
Comment (pravisankar) [inline plugins/osdn/ovs/controller.go]: If we are not setting cookie, we don't have to specify it right?
otx.AddFlow("table=8, priority=100, ...)

**Comment by pravisankar** [comment]
LGTM

**Comment by dcbw** [comment]
LGTM after ravi's comments.

File: plugins/osdn/ovs/controller.go [hunk 2]
```diff
@@ -338,9 +337,9 @@ func (plugin *ovsPlugin) AddHostSubnetRules(subnet *osapi.HostSubnet) {
 	cookie := generateCookie(subnet.HostIP)
 	otx := ovs.NewTransaction(BR)
 
-	otx.AddFlow("table=1, cookie=0x%s, priority=100, tun_src=%s, actions=goto_table:5", cookie, subnet.HostIP)
-	otx.AddFlow("table=8, cookie=0x%s, priority=100, arp, nw_dst=%s, actions=move:NXM_NX_REG0[]->NXM_NX_TUN_ID[0..31],set_field:%s->tun_dst,output:1", cookie, subnet.Subnet, subnet.HostIP)
-	otx.AddFlow("table=8, cookie=0x%s, priority=100, ip, nw_dst=%s, actions=move:NXM_NX_REG0[]->NXM_NX_TUN_ID[0..31],set_field:%s->tun_dst,output:1", cookie, subnet.Subnet, subnet.HostIP)
+	otx.AddFlow("table=1, priority=100, tun_src=%s, actions=goto_table:5", cookie, subnet.HostIP)
+	otx.AddFlow("table=8, priority=100, arp, nw_dst=%s, actions=move:NXM_NX_REG0[]->NXM_NX_TUN_ID[0..31],set_field:%s->tun_dst,output:1", cookie, subnet.Subnet, subnet.HostIP)
+	otx.AddFlow("table=8, priority=100, ip, nw_dst=%s, actions=move:NXM_NX_REG0[]->NXM_NX_TUN_ID[0..31],set_field:%s->tun_dst,output:1", cookie, subnet.Subnet, subnet.HostIP)
```
Comment (danwinship) [inline plugins/osdn/ovs/controller.go]: you need to remove the cookie from the argument lists too

**Pull request merged**